File: trunk/CharasLike/Windows/PreviewWindow.vb
Public Class PreviewWindow

    Private RenderedImage As Bitmap

    Private Sub Form4_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        ' Create the render
        RenderedImage = Canvas.RenderToTexture()
        PictureBox1.Image = RenderedImage
    End Sub

    Private Sub Form4_Move(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Move
        If EasyPosition.Visible = True Then Return

        'Dockable.CheckDocking()
    End Sub
End Class

File: trunk/Windows/PreviewWindow.cs
using System.Collections.Generic;
using System;
using System.Drawing;
using System.Diagnostics;
using Microsoft.VisualBasic;
using System.Collections;
using System.Windows.Forms;
using Paril.Windows.Forms.Docking;

namespace OpenCharas
{
	public partial class PreviewWindow : DockingWindowForm
	{
		public PreviewWindow() :
			base(Program.DockContainer)
		{
			InitializeComponent();
		}

		public Bitmap RenderedImage;

		public void Form4_Load(System.Object sender, System.EventArgs e)
		{
			// Create the render
			PictureBox1.Image = RenderedImage;
		}

		public void Form4_Move(System.Object sender, System.EventArgs e)
		{
			if (Program.canvasForm.easyPositionForm != null && Program.canvasForm.easyPositionForm.Visible == true)
				return;

			//Dockable.CheckDocking()
		}

		private void PictureBox1_Click(object sender, EventArgs e)
		{
			Close();
		}
	}
}
